lib/grpnames.gd

```diff
@@ -73,24 +93,24 @@
 ##  Remark:
 ##   The normal subgroup lattice of G does not contain
 ##   all the information needed for the normal subgroup lattice
-##   of G2 for a decomposition G = G1 x G2. However let 
+##   of G2 for a decomposition G = G1 x G2. However let
 ##   G = G1 x .. x Gr be finest direct product decomposition
 ##   then all Gi are normal subgroups in G. Thus all Gi occur in
-##   the set NormalSubgroups(G) and we may split G recursively 
+##   the set NormalSubgroups(G) and we may split G recursively
 ##   without recomputing normal subgroup lattices for factors.
 ##
 ##  Method to enumerate factorizations given the divisors:
 ##   Consider a strictly increasing chain
 ##     1  <  a1 < a2 < .. < a_n  <  A
-##   of positive divisors of an integer A. 
+##   of positive divisors of an integer A.
 ##   The task is to enumerate all pairs 1 <= i <= j <= n
 ##   such that a_i*a_j = A. This is done by
 ##
-##   i := 1;  
+##   i := 1;
 ##   j := n;
-##   while i <= j do                   
-##     while j > i and a_i*a_j > A do 
-##       j := j-1; 
+##   while i <= j do
+##     while j > i and a_i*a_j > A do
+##       j := j-1;
 ##     end while;
```

Comment:
Thank you, I would appreciate any help on this. I basically just edited the file (copied the relevant lines from my work branch into this clean branch), and only realized afterwards that there were a lot of extra whitespaces that my text editor removed automatically. So if you could help me how to resolve this easily, I would be happy to do it. 

Also, if there are any conventions on the look of the code (no. of spaces for indentation, us TAB or space, etc.), then I would gladly comply with those, as well.